pkg/gke/components/CruGKE.vue

```diff
@@ -697,7 +697,10 @@ export default defineComponent({
         <div
           class="row mb-10"
         >
-          <div class="col span-6">
+          <div
+            class="col span-6"
+            data-testid="name-ns-description-name"
```

Comment:
`name-ns-description` is a component which has this testid, it's best to give this one and below unique names

Reply:
though both could be avoided by using a selector something like this `[data-testid="crugke-form"]  .labeled-input:nth-of-type(x)`

Reply:
I prefer to keep this selector, as I'm utilizing the functions of this component in my tests (lines 62, 91, and 93), and these selectors are used across other tests that validate provisioning clusters with different cloud providers to reference the same fields: cluster name and cluster description.  I believe this avoids duplication of code.